scene/gui/flow_container.cpp

```diff
@@ -298,6 +313,11 @@ void FlowContainer::_validate_property(PropertyInfo &p_property) const {
 	if (is_fixed && p_property.name == "vertical") {
 		p_property.usage = PROPERTY_USAGE_NONE;
 	}
+	if (p_property.name == "center_alignment_wrap") {
+		if (alignment != ALIGNMENT_CENTER) {
+			p_property.usage = PROPERTY_USAGE_NO_EDITOR;
```

Comment:
`PROPERTY_USAGE_NO_EDITOR` makes the property saved in scene even if unused. I wonder if it's desired 🤔

Reply:
Seems like it should be set to hide the property when the scene is opened, but are you thinking of an alternative?

Reply:
`PROPERTY_USAGE_NONE` would make it hidden from the inspector *and* not saved in the scene. You could argue that someone might want to preconfigure `center_alignment_wrap` and then switch alignment to center at runtime, but it's unlikely.
So you could change it to NONE.

Reply:
`p_property.usage` is now set to `PROPERTY_USAGE_NONE` per the amended commit below.

Thanks!